### DISCUSSION TITLE
How to call `STRINGEQUALS` case insensitive from LINQ?

### DISCUSSION THREAD

**Discussion by xperiandri** [Azure/azure-cosmos-dotnet-v3#4534] [Q&A]
I don't see any samples, nor I can find a code that translates LINQ to `STRINGEQUALS`

**Comment by xperiandri** [comment]
@DaRosenberg

**Comment by Pilchie** [comment]
@adityasa?

**Reply by xperiandri** [reply]
Any updates?

**Comment by adityasa** [comment]
Apologies, this one slipped through.

You can supply the StringComparison parameter to Equals method with a value of StringComparison.OrdinalIgnoreCase as follows:

Expression:
query.Select(doc => doc.StringField.Equals("STR", StringComparison.OrdinalIgnoreCase))

Query:
SELECT VALUE StringEquals(root["StringField"], "STR", true) 
FROM root

Hope this helps. Adding @leminh98 as FYI.
